testing/runner/Views/Main/RunSuite.cshtml

```diff
@@ -6,7 +6,8 @@
         || Model.ScriptVirtualPath.Contains("Bundles")
         || Model.ScriptVirtualPath.Contains("DevExpress.angular")
         || Model.ScriptVirtualPath.Contains("DevExpress.jquery");
-    var isIntlTest = Model.ScriptVirtualPath.Contains("intl.tests");
+    var isIntlTest = Model.ScriptVirtualPath.Contains("intl.tests")
```

Comment:
@nightskylark @DokaRus
Can we get rid of this and use `import` or `require` for Intl in corresponding test files?

Reply:
Let's try it